symphony-bdk-core/build.gradle

```diff
@@ -72,7 +72,7 @@ dependencies {
 }
 
 // OpenAPI code generation
-def apiBaseUrl = "https://raw.githubusercontent.com/finos/symphony-api-spec/16a902e2d0f8f64c680b5d799f871a3fc4884890"
+def apiBaseUrl = "https://raw.githubusercontent.com/finos/symphony-api-spec/3171d0d55e1c1e4d1dd4931a91bf78c61d29f4d8"
```

Comment:
Be careful when merging it that last version is used (I merged my PR on the parentMessageId)